arangod/Aql/EnumerateCollectionExecutor.cpp

```diff
@@ -42,6 +42,10 @@
 
 using namespace arangodb;
 using namespace arangodb::aql;
+  
+namespace {
+std::vector<size_t> const emptyAttributePositions;
```

Comment:
is this supposed to be thread_local

Reply:
The vector is only used to be passed into the DocumentProducingContext, where a const reference to it is stored. The data in the vector is never modified by the DocumentProducingContext, nor is it modified by EnumerateCollectionExecutor(Infos).
Thus it can be static and const.